Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM docker.io/fedora:27
+
+
+LABEL ios.k8s.display-name="deepspeech" \
+    maintainer="Keith Tenzer <ktenzer@redhat.com>"
+
+RUN dnf groupinstall -y 'C Development Tools and Libraries'
+RUN dnf install -y python2 \
+    python2-devel \
+    git \
+    wget \
+    swig \
+    langpacks-de \
+    langpacks-es \
+    langpacks-fr \
+    langpacks-pl
+
+RUN mkdir /deepspeech
+
+RUN pip2 install 'deepspeech==0.3.0' \
+    jamspell \
+    webrtcvad \
+    requests \
+    certifi \
+    urllib3 \
+    idna \
+    chardet
+
+RUN cd /deepspeech && \
+    git clone https://github.com/BoseCorp/py-googletrans.git && \
+    cd /deepspeech/py-googletrans && \
+    python2 /deepspeech/py-googletrans/setup.py install
+
+RUN rm -rf /deepspeech/py-googletrans
+
+RUN wget https://github.com/mozilla/DeepSpeech/releases/download/v0.3.0/deepspeech-0.3.0-models.tar.gz -P /deepspeech && \
+    tar xzf /deepspeech/deepspeech-0.3.0-models.tar.gz -C /deepspeech
```

Comment:
You'll also want to download the jamspell English language model as shown here:

https://github.com/bakwc/JamSpell#download-models

The stream_with_sentences.py script will spell correct the English audio before sending it to the translation API in order to get better results.